src/styles/partials/components/_table.scss

```diff
@@ -13,6 +13,13 @@
     padding: $padding-large;
   }
 
+  .td_weather_details_overflow {
+    max-width: 1000px; //This is excessively large because its exclusive to the weather closing section.
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
```

Comment:
This should probably be moved into a new file and out of table to be honest. Maybe a page specific file like _closings.scss. Thoughts?

Reply:
I agree, if this doesn't apply for all tables, but just weather/closing related, it would be better in a site specific file.

Reply:
Mulling this over, so I'll have to get back to you. On one hand, it would condense table styles into one section/ document. On the other, it could be the beginnings of placing all the weather styles together.